docs/react-testing-library/setup.md

```diff
@@ -144,6 +144,35 @@ module.exports = {
 
 </details>
 
+<details>
+<summary>Modify queries globally with custom render</summary>
+
+You can override and append queries via the render function by passing a [`queries`](api.md#render-options) option.
+
+If you want to add custom queries globally, you can do this by defining a custom render method:
+
+```js
+// test-utils.js
+import { render, queries, queryHelpers } from '@testing-library/react'
+
+const customQueries = {
+  getByDataCy: queryHelpers.queryByAttribute.bind(
+    null,
+    'data-cy'
+  ),
+};
+
+const customRender = (ui, options) =>
+  render(ui, { queries: { ...queries, ...customQueries } })
+
+// re-export everything
+export * from '@testing-library/react'
+
+// override render method
+export { customRender as render }
+```
+</details>
```

Comment:
Looks great! It'd be amazing if, after the config example, we provided a demo usage (as simple as querying a single `<div>` or whatever).
Some people might have a hard time understanding the `queryHelpers`/`buildQueries` API, so a final example showing how to use the newly availalbe `getByDataCy` would come in handy. What do you think about that?

Reply:
Fair point! Added a `getByDataCy` usage example 👍